gno.land/cmd/gnoland/testdata/maketx_call_sponsor.txtar

```diff
@@ -0,0 +1,26 @@
+# test for maketx call with sponsorship
+loadpkg gno.land/r/foo/bar $WORK
+
+# start a new node
+gnoland start
+
```

Comment:
the test should demonstrate that test2 isn't existing yet onchain

Reply:
https://github.com/gnolang/gno/pull/2630/commits/dec14a8425ea38e40653849004d4ca969fb14194